### PR TITLE
Fix saved build

### LINKED ISSUE #73: Wrong build value in genes, transcripts, exons
It should be "GRCh37", not "build_37"

I must have introduced this error in one of my latest PRs:

image

image

image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #73 

### How to test:
- Load genes, transcripts and exons in the database

### Expected outcome:
- [x] Start the app and make sure that genes, transcripts and exons tables contain the right build format
- [x] Run a query on each of these tables and make sure the build has the expected format

### Review:
- [ ] Code approved by
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Main branch: the tables contain wrong values for the build enum in all 3 tables:

image

The format for the enum is correct when initializing the database using this branch: 🆗 

image

**Comment by northwestwitch** [comment]
Items are saved correctly using this branch. Genes:

image

Transcripts:

image

Exons:

image

File: src/chanjo2/models/pydantic_models.py [hunk 1]
```diff
@@ -12,6 +12,10 @@ class Builds(str, Enum):
     build_37 = "GRCh37"
     build_38 = "GRCh38"
 
+    @staticmethod
+    def get_enum_values() -> List[str]:
+        return [member.value for member in Builds]
```
Comment (henrikstranneheim) [inline src/chanjo2/models/pydantic_models.py:17]: Add doc str

**Review by henrikstranneheim — APPROVED** [review]
Nice 👍

**Pull request merged**